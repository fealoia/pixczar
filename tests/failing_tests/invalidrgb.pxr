Void main() {
  Pix test = new Pix();
  test.makeTriangle(200, [])
}