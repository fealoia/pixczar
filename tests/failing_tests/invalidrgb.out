Fatal error: exception Failure("Invalid rgb input")
